### PR TITLE
[M1952] bugfix: fish with no regions

### PR DESCRIPTION
**Pull request by saanobhaai**
## Release Notes

* **New Features**
  * Added new management command to import and update classifier attributes from CSV files with dry-run support.

* **Bug Fixes**
  * Fixed region validation to prevent crashes when processing records with incomplete region data.

* **Tests**
  * Added test coverage for region validation with edge-case data scenarios.

[Review Change Stack](https://app.coderabbit.ai/change-stack/data-mermaid/mermaid-api/pull/681)

### CODE REVIEW

**Comment by saanobhaai** [comment]
[500 error for fish with missing regions](https://trello.com/c/S7xKTovt/1952-500-error-for-fish-with-missing-regions)

File: src/tools/management/commands/set_classifier_bagfs.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        try:
+            classifier = Classifier.objects.get(pk=classifier_id)
+        except Classifier.DoesNotExist:
+            raise CommandError(f"Classifier {classifier_id} not found.")
+
+        pairs = []
+        with open(csv_path, newline="") as f:
+            for row in csv.DictReader(f):
+                ba_id = row["benthic_attribute_id"].strip()
+                gf_id = row["growth_form_id"].strip() or None
```
[the remark this answers is not in the document]
Comment (saanobhaai) [reply src/tools/management/commands/set_classifier_bagfs.py:41]: This is an admin CLI command run once a year; the crash would actually be desirable. Don't fix.

**Review by gridcell — APPROVED** [review]
